packages/core/src/components/button/_button-group.scss

```diff
@@ -139,7 +139,7 @@ Styleguide pt-button-group
       }
     }
 
-    > .pt-popover-target:last-child .pt-button,
+    > .pt-popover-wrapper:last-child .pt-popover-target .pt-button,
```

Comment:
sad long selector 😢

Reply:
I know. I've had to `stylelint-disable` that rule quite a bit here.